### ISSUE TITLE
Separate Explorer's FileSystem and Outline View

### ISSUE THREAD

**Issue by jelmd** [microsoft/vscode#181205]
Right now the Explorer View combines the File system view and OUTLINE view. This is the most annoying thing ever seen. During development it really sucks hardly, because if both are open, they are far too small to work efficiently with it, i.e. show enough lines to have an usable overview and one does not always need to scroll miles and search. Making decreasing the font size makes UI experience even worse, is no workaround (IMHO the default is already far to small and stresses the developers eyes in a way, tat decreases their productivity), i.e. current explorer UI is inconvenient, annoying, exhausting and unhealthy.

So please make OUTLINE a separate view (similar to the search, explorer, source control view), to be able to use the full height of the related editor panel and select it with a single click on a sidebar icon.

In case a visual idea is needed: Just have a look at Eclipse: it solved this properly by providing both as separate "standalone" views, i.e. one just has to click on the related tab header and can switch over to it.

**Comment by gjsjohnmurray** [comment]
Sounds like you aren't aware that you can drag Outline to the opposite side and drop it into the secondary sidebar that lives there.

**Comment by jelmd** [comment]
Correct. Tried it - but it wastes to much space another 20-30% of width - not good. Clicked on the title bar's X (close) and than it took me some minutes to get it back. Anyway, after that I noticed the small symbol in its upper left and that it seems to be draggable -  dragged it to the icon bar and surprise, surprise - it worked. So now I have, what I actually was asking for :-)

Perhaps having the icon in place on the title bar even when it is shown in the explorer window, as well as getting the drag icon, if one click and holds on the title bar even on empty areas would have helped me to eventually figure it out by myself.

However, thanx a lot for the hint!
